play/src/components/ConfidenceMeterCard.tsx

```diff
@@ -0,0 +1,181 @@
+  const confidenceOptionArray = [
+    {
+      text: t('gameinprogress.chooseanswer.confidenceoption1'),
+      value: ConfidenceLevel.NOT_AT_ALL,
+    },
+    {
+      text: t('gameinprogress.chooseanswer.confidenceoption2'),
+      value: ConfidenceLevel.KINDA,
+    },
+    {
+      text: t('gameinprogress.chooseanswer.confidenceoption3'),
+      value: ConfidenceLevel.QUITE,
+    },
+    {
+      text: t('gameinprogress.chooseanswer.confidenceoption4'),
+      value: ConfidenceLevel.VERY,
+    },
+    {
+      text: t('gameinprogress.chooseanswer.confidenceoption5'),
+      value: ConfidenceLevel.TOTALLY,
+    },
+  ];
+
+  const confidenceHeader = (
+    <Box
+      display="inline"
+      sx={{
+        textAlign: 'left',
+        marginX: `${theme.sizing.extraSmallPadding}px`,
+        width: '100%',
+        marginBottom: `${theme.sizing.smallPadding}px`,
+      }}
+    >
+      <Typography
+        variant="h2"
+        sx={{ color: `${theme.palette.primary.darkBlue}` }}
+      >
+        {t('gameinprogress.chooseanswer.confidenceheader')}
+      </Typography>
+    </Box>
+  );
+  const chooseConfidenceText = (
+    <Box display="inline" sx={{ textAlign: 'center' }}>
+      <Typography variant="body1">
+        {t('gameinprogress.chooseanswer.answerthankyou1')}
+      </Typography>
+      <Typography variant="body1">
+        {t('gameinprogress.chooseanswer.confidencetext')}
+      </Typography>
+    </Box>
+  );
+  const responseOption = (text: string, index: number) => {
+    return (
+      <Box
+        maxWidth={`${theme.sizing.extraLargePadding}px`}
+        sx={{
+          textAlign: 'center',
+          alignItems: 'center',
+          marginX: `${theme.sizing.extraSmallPadding}px`,
+        }}
+      >
+        <FormControlLabel
+          key={index}
+          value={index}
+          control={
+            <Radio
+              size={isSmallDevice ? 'small' : 'medium'}
+              sx={{
+                '&.Mui-checked': {
+                  color: `${theme.palette.primary.blue}`,
+                },
+              }}
+            />
+          }
+          label={text}
+          labelPlacement="bottom"
+          sx={{ marginX: '0' }}
+        />
+      </Box>
+    );
+  };
+  const sendingResponseText = (
+    <Box
+      display="inline"
+      sx={{
+        textAlign: 'center',
+        marginBottom: `${theme.sizing.smallPadding}px`,
+      }}
+    >
+      <Typography
+        variant="body1"
+        sx={{ color: `${theme.palette.primary.darkBlue}`, opacity: '0.5' }}
+      >
+        {isSelected
+          ? t('gameinprogress.chooseanswer.submittedconfidenceresponse')
+          : t('gameinprogress.chooseanswer.sendingconfidenceresponse')}
+      </Typography>
+    </Box>
+  );
+
+  const handleRadioChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const currentTime = new Date().getTime() / 1000;
+    if (
+      isNullOrUndefined(timeOfLastSelect) ||
+      currentTime - timeOfLastSelect > 5
```

Comment:
Usually when working with time, it'd be nice to comment on what intention is. This helps others checking the code to know what the interval is. One other way can be having the code documents itself:

```typescript
let currentEpochTimeInSeconds = new Date().getTime() / 1000;
```